WooCommerce/Resources/Images.xcassets/POS/pos-reader-connection-complete.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "pos-reader-connection-complete.png",
```

Comment:
Question: We currently have a mix of `png` and `pdf` images in `/Resources`, I'm not sure if it's worth to use `pdf` everywhere as it seems to require less space? Or this would mess with the transparency?

Reply:
In my past experience the PDFs were much bigger, but that's been improved, looking at more recent images.

I think it was the images which combined vector and raster images, which became massive as PDFs – it's less of an issue with these simpler images. I'll swap them over.